webpack.config.js

```diff
@@ -94,7 +93,8 @@ const webpackConfig = {
 		new webpack.DefinePlugin( {
 			'process.env': {
 				NODE_ENV: JSON.stringify( bundleEnv )
-			}
+			},
+			'PROJECT_NAME': JSON.stringify( config( 'project' ) )
```

Comment:
This should be in `webpack.config.node.js` too, it should work there? Don't override `process.env` though :)